novawallet/Modules/Vote/Governance/Operation/Fetch/OffchainVoting/SubqueryVotingOperationFactory.swift

```diff
@@ -91,6 +91,25 @@ final class SubqueryVotingOperationFactory: SubqueryBaseOperationFactory {
         """
     }
 
+    private func prepareSplitAbstainVotesQuery(referendumId: ReferendumIdLocal) -> String {
```

Comment:
I would make requests to votes (ReferendumDetails screen) and voters (Voters screen when we tap on Abstain votes on the details) different. Those requests will have the same filter but different nodes. Votes request will have the same nodes as we have now and voters will have the same nodes as ```prepareReferendumVotersQuery```